app/api/events.py

```diff
@@ -104,6 +105,14 @@ class Meta:
                                  schema='EventInvoiceSchema',
                                  many=True,
                                  type_='event_invoice')
+    discount_codes = Relationship(attribute='discount_code',
```

Comment:
Shouldn't this be `discount_code`  since event model has one `discount_code_id` ... So one-to-one ?